src/unpkg-plugin/UnpkgPlugin.js

```diff
@@ -0,0 +1,13 @@
+require('ts-node').register({
+  projectSearchDir: __dirname,
+});
+const { UnpkgPlugin } = require('./UnpkgPluginImpl');
+
+module.exports = function (PluginHost) {
```

Comment:
Nooooooooooo

1. `module.exports =` style plugins are deprecated, use `exports.load = function` instead
2. PluginHost is dead. Plugins are now passed an `Application` instance
3. `addComponent` is going to go away. The whole idea of components overcomplicates everything. Plugins should never have used the `@Component` decorator.

Reply:
Ok, this was copy-pasted from https://github.com/gdelmas/typedoc-plugin-sourcefile-url